tinyosc.c

```diff
@@ -301,8 +302,8 @@ void tosc_printMessage(tosc_message *osc) {
       case 'f': printf(" %g", tosc_getNextFloat(osc)); break;
       case 'd': printf(" %g", tosc_getNextDouble(osc)); break;
       case 'i': printf(" %d", tosc_getNextInt32(osc)); break;
-      case 'h': printf(" %lld", tosc_getNextInt64(osc)); break;
-      case 't': printf(" %lld", tosc_getNextTimetag(osc)); break;
+      case 'h': printf(" %" PRId64, tosc_getNextInt64(osc)); break;  // PRId64 chooses corrent format (ld,lld,..)
```

Comment:
What kind of application do you have such that "old compilers" (which apparently don't support C99) are necessary?